scripts/playbook.yml

```diff
@@ -29,11 +31,15 @@
         - cmake
         - python-psycopg2
         - libselinux-python
-        - git
         - bash-completion
         - vim
-    - name: install bower package manager
-      npm: name=bower global=yes
+    - name: install npm packages needed
+      npm: name={{ item }} global=yes
+      with_items:
+        - bower
+        - yarn
+        - gulp-cli
+        - webpack
```

Comment:
right, packages needed are just bower and yarn as per [ui build kickstart partial](https://github.com/ManageIQ/manageiq-appliance-build/blob/ed659aa35bfc70be9c255eeed467ff45350ede46/kickstarts/partials/post/ui_compile.ks.erb#L1)